site/content/api/Telemetry-API.md

```diff
@@ -63,7 +63,7 @@ The following table provides an overview of the properties a client needs to set
 | Name            | Mandatory | Location                 | Type      | Description |
 | :-------------- | :-------: | :----------------------- | :-------- | :---------- |
 | *content-type*  | yes       | *properties*             | *symbol*  | SHOULD be set to *application/octet-stream* if the message payload is to be considered *opaque* binary data. In most cases though, the client should be able to set a more specific content type indicating the type and characteristics of the data contained in the payload, e.g. `text/plain; charset="utf-8"` for a text message or `application/json` etc. |
-| *creation-time* | yes       | *properties*             | *timestamp* | MUST be set to the current number of milliseconds since the unix epoch (1970-01-01T00:00:00Z). |
+| *creation-time* | yes       | *properties*             | *timestamp* | the time when this message was created (see the [AMQP 1.0 specification](http://docs.oasis-open.org/amqp/core/v1.0/amqp-core-messaging-v1.0.html) for details).
```

Comment:
FMPOV this is mandatory only if ttd is set, isn't it?

Reply:
That is true, although the protocol adapters now always set it.
But since this is the specification, I completely agree - will adopt the text.